scripts/install

```diff
@@ -1,202 +1,234 @@
 #!/bin/bash
 
-# Active trap pour arrêter le script si une erreur est détectée.
-EXIT_PROPERLY () {      # Provoque l'arrêt du script en cas d'erreur. Et nettoye les résidus.
-        exit_code=$?
-        if [ "$exit_code" -eq 0 ]; then
-                        exit 0  # Quitte sans erreur si le script se termine correctement.
-        fi
-        trap '' EXIT
-        set +eu
-        echo -e "\e[91m \e[1m"  # Shell in light red bold
-        echo -e "!!\n  $app install's script has encountered an error. Installation was cancelled.\n!!" >&2
-
-        if type -t CLEAN_SETUP > /dev/null; then        # Vérifie l'existance de la fonction avant de l'exécuter.
-                CLEAN_SETUP     # Appel la fonction de nettoyage spécifique du script install.
-        fi
-
-        # Compense le bug de ssowat qui ne supprime pas l'entrée de l'app en cas d'erreur d'installation.
-        sudo sed -i "\@\"$domain$path/\":@d" /etc/ssowat/conf.json
-
-        ynh_die
-}
-
-TRAP_ON () {    # Activate signal capture
-        set -eu # Exit if a command fail, and if a variable is used unset.
-        trap EXIT_PROPERLY EXIT # Capturing exit signals on shell script
-}
-
-TRAP_ON
-
-# Source app helpers
+#=================================================
+# GENERIC START
+#=================================================
+# IMPORT GENERIC HELPERS
+#=================================================
+
+source _common.sh
 source /usr/share/yunohost/helpers
 
-# Retrieve arguments
-app=$YNH_APP_INSTANCE_NAME
+#=================================================
+# MANAGE SCRIPT FAILURE
+#=================================================
+
+# Exit if an error occurs during the execution of the script
+ynh_abort_if_errors
+
+#=================================================
+# RETRIEVE ARGUMENTS FROM THE MANIFEST
+#=================================================
+
 domain=$YNH_APP_ARG_DOMAIN
-path=$YNH_APP_ARG_PATH
 
-# We check variables are not empty
-CHECK_VAR () {  # Vérifie que la variable n'est pas vide.
-# $1 = Variable à vérifier
-# $2 = Texte à afficher en cas d'erreur
-        test -n "$1" || (echo "$2" >&2 && false)
-}
+path="/Microsoft-Server-ActiveSync"
```

Comment:
If possible, use `path_url` instead of `path`

Reply:
Switch to path_url done.
I haven't done before because I didn't want to make variable name changes ^^